### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by jd-lara**

### CODE REVIEW

File: docs/src/quick_start_guide.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Quick Start Guide
+
+You can access example data in the [Power Systems Test Data Repository](https://github.com/NREL-SIIP/PowerSystemsTestData),
+the data can be downloaded with the `PowerSystems.jl` submodule `UtilsData`
```
Comment (rodrigomha) [inline docs/src/quick_start_guide.md:4]: add a period here.

File: docs/src/quick_start_guide.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+```@repl quick_start_guide
+using PowerSystems, PowerSimulationsDynamics, Sundials, Plots
+DATA_DIR = download(PowerSystems.UtilsData.TestData, folder = pwd())
+omib_sys = System(joinpath(DATA_DIR, "psse_raw/OMIB.raw"),
+                     joinpath(DATA_DIR, "psse_raw/OMIB.dyr"))
+```
+
+For more details about loading data and adding more dynamic components check the
+[Creating a System with Dynamic devices](https://nrel-siip.github.io/PowerSystems.jl/stable/modeler_guide/system_dynamic_data/)
+section of the documentation in `PowerSystems.jl`
```
Comment (rodrigomha) [inline docs/src/quick_start_guide.md:19]: a period here.

File: docs/src/quick_start_guide.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Loading data
+
+Data can be loaded from a pss/e raw file and a pss/e dyr file.
+
+```@repl quick_start_guide
+using PowerSystems, PowerSimulationsDynamics, Sundials, Plots
+DATA_DIR = download(PowerSystems.UtilsData.TestData, folder = pwd())
+omib_sys = System(joinpath(DATA_DIR, "psse_raw/OMIB.raw"),
+                     joinpath(DATA_DIR, "psse_raw/OMIB.dyr"))
```
Comment (rodrigomha) [inline docs/src/quick_start_guide.md:14]: Is not creating the system properly I think. Check download reference?

**Review by rodrigomha — COMMENTED** [review]
In the file tutorials_page.md there are some typos. Add a period after the first paragraph. There is a typo `Infite` that should say `Infinite`

File: docs/src/reference_frames.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Note that hence in a bus of ``1.0\angle 0``, a rotor angle of ``\delta = 0`` implies that
+``v_q = 1.0`` and ``v_d = 0.0``. This transformation is the one that can be found in most
+books of Power Systems, such as Kundur, Sauer Pai and in Milano too, and is the convention
+used in the software to model dynamic models of synchronous machines in their own reference
+frame.
+
+## Inverters
+
+The previously convention is not the standard one used for modeling inverters. Most of
```
Comment (rodrigomha) [inline docs/src/reference_frames.md:31]: previous instead of previously

File: docs/src/models.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+- Machine: That defines the stator electro-magnetic dynamics.
+- Shaft: That describes the rotor electro-mechanical dynamics.
+- Automatic Voltage Regulator: Electromotive dynamics to model an AVR controller.
+- Power System Stabilizer: Control dynamics to define an stabilization signal for the AVR.
+- Prime Mover and Turbine Governor: Thermo-mechanical dynamics and associated controllers.
+
+The following figure summarizes the components of a generator and which variables they share:
+
+![gen_meta](https://github.com/NREL-SIIP/PowerSystems.jl/blob/master/docs/src/assets/gen_metamodel.png)
```
Comment (rodrigomha) [inline docs/src/models.md:16]: Check reference of the file since it is not being displayed

File: docs/src/models.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+- DC Source: Defines the dynamics of the DC side of the converter.
+- Frequency Estimator: That describes how the frequency of the grid can be estimated using the grid voltages. Typically a phase-locked loop (PLL).
+- Outer Loop Control: That describes the active and reactive power control dynamics.
+- Inner Loop Control: That can describe virtual impedance, voltage control and current control dynamics.
+- Converter: That describes the dynamics of the pulse width modulation (PWM) or space vector modulation (SVM).
+- Filter: Used to connect the converter output to the grid.
+
+The following figure summarizes the components of a inverter and which variables they share:
+
+![inv_meta](https://github.com/NREL-SIIP/PowerSystems.jl/blob/master/docs/src/assets/inv_metamodel.png)
```
Comment (rodrigomha) [inline docs/src/models.md:35]: same here for file reference

File: docs/src/component_models/machines.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+and for the GENROE model the function used is:
+
+```math
+\begin{align}
+\text{Se}(\psi'') &= B(\psi'')^A \tag{7r}
+\end{align}
+```
+
+The parameters ``A`` and ``B`` for each function are computed using the two points given
+``(1.0, \text{Se}(1.0))`` and ``(1.2, \text{Se}(1.2))``.
```
Comment (rodrigomha) [inline docs/src/component_models/machines.md:163]: I will add GENSAL

File: docs/src/component_models/avr.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+```
+
+with
+
+```math
+\begin{align*}
+v_r &= K_0v_{r2} + \frac{T_4}{T_3} \left( v_{r1} + K_0\frac{T_2}{T_1}(v_{\text{ref}} - v_m)\right) \\
+S_e(v_f) &= A_e \exp(B_e|v_f|)
+\end{align*}
+```
```
Comment (rodrigomha) [inline docs/src/component_models/avr.md:60]: I will add AC1A

File: docs/src/component_models/turbine_gov.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+### TG Type II ```[TGTypeII]```
+
+This turbine governor is a simplified model of the Type I.
+
+```math
+\begin{align}
+\dot{x}_g &= \frac{1}{T_2}\left[\frac{1}{R}\left(1 - \frac{T_1}{T_2}\right) (\omega_s - \omega) - x_g\right] \tag{14a} \\
+\tau_m &= P_{\text{ref}} + \frac{1}{R}\frac{T_1}{T_2}(\omega_s - \omega) \tag{14b}
+\end{align}
+```
```
Comment (rodrigomha) [inline docs/src/component_models/turbine_gov.md:40]: I will add TGOV1 and GAST

File: docs/src/api/public.md [hunk 10]
```diff
@@ -7,28 +7,8 @@ DocTestSetup  = quote
...
-
-```@index
-Pages = ["PowerSimulationsDynamics.md"]
-```
-
-## Exported
-
 ```@autodocs
 Modules = [PowerSimulationsDynamics]
+Public = true
```
Comment (rodrigomha) [inline docs/src/api/public.md:12]: There is an error in the docstring of print_device_states. It only receives the simulation.

File: docs/src/api/internal.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+```@meta
+CurrentModule = PowerSimulationsDynamics
+DocTestSetup  = quote
+    using PowerSimulationsDynamics
+end
+```
+
+```@autodocs
+Modules = [PowerSimulationsDynamics]
+Public = false
```
Comment (rodrigomha) [inline docs/src/api/internal.md:12]: There are some errors in our docstrings. The `inverter_inner_vars` is not being properly printed.

Comment (jd-lara) [reply docs/src/api/internal.md:12]: Not sure about this. I had to disable DocumeterExtensions since some of the functions signatures failed.

**Pull request merged**